includes/class-wc-tracker.php

```diff
@@ -104,6 +104,13 @@ private static function get_tracking_data() {
 		$data['active_plugins']     = $all_plugins['active_plugins'];
 		$data['inactive_plugins']   = $all_plugins['inactive_plugins'];
 
+		// Jetpack & WooCommerce Connect
+		$data['jetpack_version']    = defined( 'JETPACK__VERSION' ) ? JETPACK__VERSION : 'none';
+		$data['jetpack_connected']  = ( class_exists( 'Jetpack' ) && method_exists( 'Jetpack', 'is_active' ) && Jetpack::is_active() ) ? 'yes' : 'no';
```

Comment:
Replace `method_exists` with `is_callable` - a method may exist, needs public scope to be used here.